RELEASE-NOTES.md

```diff
@@ -1,5 +1,73 @@
 # Cumulative Release Notes for the Annotated Data Model
 
+## ???
+
+### [ROS-218](https://basistech.atlassian.net/browse/ROS-218) Entity-level sentiment is a list
+
+Entity-level sentiment used to return a single `CategorizerResult`.
+After this change it returns a `List<CategorizerResult>` in confidence
+ranked order.  Currently there will be three results, one for each of
+"neg", "neu", "pos".  `ResolvedEntity#getSentiment` is already
+deprecated because of ROS-43.  It still returns the top result only.
+`Entity#getSentiment()` is the new method that returns the list.
+
+### [ROS-43](https://basistech.atlassian.net/browse/ROS-43) Combine EntityMentions and ResolvedEntity
```

Comment:
@benson-basis - you should probably review my release notes for ROS-43 and ROS-50.